yake_rust/src/stopwords/mod.rs

```diff
@@ -18,7 +18,7 @@ impl StopWords {
     }
 
     /// Load a predefined list of stopwords for the language given as argument.
-    pub fn predefined(lang: &str) -> Option<Self> {
+    pub fn predefined(lang: &str) -> Result<Self, String> {
```

Comment:
Don't you mind keeping the interface intact as we also provide python bindings? You could do the same through an adapter function anyways

Reply:
Fair enough. Will fix.